### DISCUSSION TITLE
Improve CI / publishing

### DISCUSSION THREAD

**Discussion by filipef101** [gbumps/react-native-screenguard#74] [Ideas]
Think this all can be done independently:
1. Remove types/commonjs from repo
Doing the building before publishing
2. Let CI handle the publishing, ie when creating a release on GH
3. CI to validate linting and tsc on commit (to sanity check PRs)

Worth checking out [react-native-builder-bob](https://callstack.github.io/react-native-builder-bob/create) either to use it or copy some inspiration for actions

I'm happy to take a jab at that if maintainer agrees and accepts

**Comment by gbumps** [comment]
Just create a PR and I'll carefully look into it.
